HLTrigger/HLTfilters/plugins/L1TTkMuonFilter.cc

```diff
@@ -26,6 +27,31 @@
 // constructors and destructor
 //
 
+namespace {
+  bool passMuQual(const l1t::TkMuon& muon) {
+    if (muon.muonDetector() == 3) {
+      int quality = muon.quality();
+      if (quality == 11 || quality == 13 || quality == 14 || quality == 15) {
```

Comment:
are these values defined anywhere, *e.g.* in some L1T package ?

Reply:
no idea. I freely admit these magic numbers are not great and should move. What we could do is make it configurable with a vector "allowed qualities". 

Basically I read the twiki and applied the cuts? That is extent of my understanding here :)

Reply:
Indeed, I haven't been able to find any definition that one could reuse in the L1 software